tests/unit/static/golden/api.trulens_eval.3.11.yaml

```diff
@@ -1,6 +1,6 @@
 trulens_eval:
   __class__: builtins.module
-  __version__: 0.33.0
+  __version__: 1.0.0.a
```

Comment:
these are expected to stay static. We need to update src/trulens_eval to match the <1.0 spec though. See note above about aliasing the import